src/index.md

```diff
@@ -218,11 +220,7 @@ Inputs.table(storage_providers, {
   rows: 20,
   select: false,
   format: {
-    provider_id: id => htl.html`<a href=/filecoin-storage-providers-market/provider/${id} target=_blank>${id}</a>`
+    provider_id: id => htl.html`<a href=./provider/${id} target=_blank>${id}</a>`
```

Comment:
I wasn't sure how to do this! :sweat_smile: I even tried making a variable that checked if it was running on `localhost`. The solution was much simpler, as it always happens.